src/routes/businessClass.js

```diff
@@ -21,19 +20,16 @@ const {
 const { getAllBusinessSession } = require("../controllers/businessSession");
 const { isAuthorized } = require("../middlewares/auth");
 const { getAllTermsInAClass } = require("../controllers/term");
+const { CLASS_DEFINITION } = require("../constants/pages");
+const getResourceBusinessIdInCreate = require("../middlewares/auth/utils/getResourceBusinessIdInCreate");
+const getResourceBusinessIdInUpdate = require("../middlewares/auth/utils/getResourceBusinessIdInUpdate");
+const isAuthHandler = require("../middlewares/auth/utils/isAuthHandler");
 
 //parameters
 // router.param("businessClassId", getBusinessClassIdById);
 
 //all of actual routes
 
-const isAuthHandler = (req, res) => {
-  /**
-   * TODO: add the logic.
-   */
-  return true;
-};
-
 router.get(
   "/of-logged-in-user",
   isAuthorized(null, null, { isAuthHandler }),
```

Comment:
this route is only for business admin. please remove the is 3rd parameter, it should be isAuthorized(CLASS_DEFINITION, READ, { getResourceBusinessId }).
Note: Initially the route was planned for parents as well, However Now it's only for business admin.

Reply:
i have texted u regarding this in backend grp please check

Reply:
hi we cannot have authHandler for this route . there is nothing to compare i.e. as its a get route we are not having any body and as it is not having any params also ,we are directly fetching the user data from token and validating it in controller itself soo there is noo need of any restrictions for this route as it is already implemented in the controller itself

Reply:
we need to add isAuthorized(null, null), so that we can use the following code in controller `let { authUserData } = req;`

Reply:
ok done